content/docs/get-started/install/_index.md

```diff
@@ -171,6 +171,66 @@ or visit https://pulumi.com/docs/reference/install/ for manual instructions and
 
 If you are upgrading to Pulumi 2.0, please see our [migration guide]({{< relref "migrating-2.0" >}}).
 
+## Installing Previous Versions
+
+You can find the list of versions on the [Available Versions]({{< relref "/docs/get-started/install/versions" >}}) page.
+
+{{< chooser os "macos,windows,linux" >}}
+
+{{% choosable os macos %}}
+
+### Homebrew
+
+```bash
+$ brew install pulumi@<version>
+```
+
+### Installation Script
+
+```bash
+$ curl -fsSL https://get.pulumi.com | sh -s -- --version <version>
```

Comment:
The double dashes (`--`) get rendered as single dashes (-) when published and as such don't work when copy/pasted and used as single dashes.

from http://pulumi-docs-origin-pr-4200-23d76ac1.s3-website.us-west-2.amazonaws.com/docs/get-started/install/#installing-previous-versions:

![image](https://user-images.githubusercontent.com/293763/93164849-6c11f200-f6cf-11ea-8d6d-50f20b71e8be.png)

Reply:
CLI output with single dashes and double dashes

```
root@f69176c035e3:/# curl -fsSL https://get.pulumi.com | sh -s - -version 2.2.0
=== Installing Pulumi v2.10.0 ===
+ Downloading https://get.pulumi.com/releases/sdk/pulumi-v2.10.0-linux-x64.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
 72 61.2M   72 44.1M    0     0  11.8M      0  0:00:05  0:00:03  0:00:02 11.8M^C
root@f69176c035e3:/# curl -fsSL https://get.pulumi.com | sh -s -- --version 2.2.0
=== Installing Pulumi v2.2.0 ===
+ Downloading https://get.pulumi.com/releases/sdk/pulumi-v2.2.0-linux-x64.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 57.2M  100 57.2M    0     0  7337k      0  0:00:07  0:00:07 --:--:-- 7043k
+ Extracting to /root/.pulumi/bin

=== Pulumi is now installed! 🍹 ===
+ Please add /root/.pulumi/bin to your $PATH
+ Get started with Pulumi: https://www.pulumi.com/docs/quickstart
```